packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -159,3 +161,13 @@ config:
       chain_id: ${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_CHAIN_ID:int:31337}
       poa_chain: ${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_POA_CHAIN:bool:false}
       default_gas_price_strategy: ${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_DEFAULT_GAS_PRICE_STRATEGY:str:eip1559}
+---
+public_id: fetchai/http_server:0.22.0
+type: connection
+config:
+  api_spec_path: null
+  host: ${HTTP_SERVER_LISTEN_ADDRESS:str:0.0.0.0}
+  port: ${HTTP_SERVER_LISTEN_PORT:int:8000}
```

Comment:
Not sure this can be overridden from the service.

Reply:
anyway, lets stay as is, it's better to have option to configure

Reply:
So what I am saying is because you have set the env vars as `HTTP_SERVER_LISTEN_ADDRESS` and `HTTP_SERVER_LISTEN_PORT` you cannot override them on the service level. If you think that is not necessary, we can leave it like this. I assume since this is meant to be deployed as a docker container (via docker-compose or k8s) multiple agents mapping to the same port will not be an issue for the service.